okhttp/src/main/kotlin/com/connectrpc/okhttp/OkHttpStream.kt

```diff
@@ -61,7 +61,7 @@ internal fun OkHttpClient.initializeStream(
     }
     val callRequest = builder.build()
     val call = newCall(callRequest)
-    call.enqueue(ResponseCallback(onResult, isSendClosed))
+    call.enqueue(ResponseCallback(onResult, isReceiveClosed))
```

Comment:
This was the other big error for client streaming calls - this was using `isSendClosed` instead of `isReceiveClosed`, so we would stop trying to read messages from a stream once the send side was closed.